src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsOnsetViewController.swift

```diff
@@ -18,9 +18,11 @@
 import UIKit
 import Combine
 
-class ExposureSubmissionSymptomsOnsetViewController: DynamicTableViewController, ENANavigationControllerWithFooterChild {
+class ExposureSubmissionSymptomsOnsetViewController: DynamicTableViewController, ENANavigationControllerWithFooterChild, RequiresDismissConfirmation {
 
-	enum SymptomsOnsetSelectionOption {
+	typealias PrimaryButtonHandler = (SymptomsOnsetOption) -> Void
+
+	enum SymptomsOnsetOption {
```

Comment:
What is the reason for you adding this information into the coordinator? Can you store this information somewhere else, preferably not the coordinator component?

Reply:
This is in the view controller, I'm not sure what you mean here 😅

Reply:
That's weird. I could have bet I put this comment someplace else. Whatever, it is fine :-)